### ISSUE TITLE
Magnum 357 has a different look

### ISSUE THREAD

**Issue by sfkylcnky** [TheGameCreators/GameGuruRepo#751]
Magnum 357 at the Weapon folder had a different look before. Is this a bug or a change made in purpose? Before it seemed more realistic and metallic. Now it looks like a plastic. Is it just me having this opinion? This change caused by its relevant dds located at 
gamecore\guns\modern\Magnum357\gun_color.dds

![whichoneiscool](https://user-images.githubusercontent.com/37545408/78683013-6a115f00-78f7-11ea-9814-bd7d78ebc105.jpg)

upper images belong to older textures and lowers are from the new.

**Comment by LeeBamberTGC** [comment]
Yes, this was done deliberately based on feedback from an artist way back when, making the weapons more PBR aligned, which did cause the weapon to change how it renders.  If you rename the various files, you could resurrect the old weapon style if you prefer the old one from the textures you refer to above.
